feature_server/commands.py

```diff
@@ -197,10 +213,16 @@ def say(connection, *arg):
     connection.protocol.send_chat(value)
     connection.protocol.irc_say(value)
 
-@admin
-def kill(connection, value):
+add_rights('kill', 'admin')
+def kill(connection, value = None):
+    if value is None:
+        player = connection
+    else:
+        if not connection.rights.kill:
+            return "You can't use this command"
     player = get_player(connection.protocol, value, False)
     player.kill()
+    if connection is not player:
```

Comment:
lesson learnt: 'rebase ignore space change' and python are not compatible. Yes, there are quite a few places where this holds true and their are screwed in terms of indentation.


Those are the places where there are changes I believe. There will be no places where it will have silently screwed, there gotta be change there (because we are comparing against our prestine `master`)